Specs/APBabyPlayer/0.1.0/APBabyPlayer.podspec

```diff
@@ -0,0 +1,41 @@
+Pod::Spec.new do |s|
+  s.name             = "APBabyPlayer"
+  s.version           = '0.1.0'
```

Comment:
Please align with the old versioning.
If the old version was 5.6 please use 6.0.